components/transactions-list.tsx

```diff
@@ -31,7 +31,9 @@ import { TransactionTypes } from "interfaces/enums/transaction-types";
 import { Transaction } from "interfaces/transaction";
 
 export default function TransactionsList({
-  onActiveTransaction = (transaction) => {}
+  setActiveTransaction
+}: {
+  setActiveTransaction: React.Dispatch<React.SetStateAction<Transaction>>
```

Comment:
this event (`onActiveTransactionChange`) should not have a default type of a Dispatch, as we might want to do _something else_ with it before calling a dispatcher. this should simply be a `: void` [please, rename to `onActiveTransactionChange`]